### PR TITLE
Model multiple orgs correctly

### PR DESCRIPTION
**Pull request by elenatanasoiu**
For:

https://trello.com/c/y1E9av7z/273-model-dit-correctly-within-signon
https://trello.com/c/lGjMWdHM/275-model-fco-correctly-within-signon
https://trello.com/c/8MA3tUOe/276-model-fsa-correctly-within-signon
https://trello.com/c/bCAm6Vkh/277-model-hmt-correctly-within-signon
https://trello.com/c/FmZ6gAFV/279-model-moj-correctly-within-signon
https://trello.com/c/G7C9Y0hl/280-model-nio-and-scotland-office-correctly-within-signon
https://trello.com/c/JWVgB511/284-model-ukti-correctly-within-signon
https://trello.com/c/0UT6Ewmd/282-model-welsh-gov-correctly-within-signon

Similar to https://github.com/alphagov/signon/pull/583, we're making sure that these organisations have their organisational structure modelled correctly (based on the hierarchy in whitehall).

For some cases where an organisation has two parents in whitehall, we've decided which parent is the "primary" parent because signon only supports a single parent structure.

### CODE REVIEW

File: db/migrate/20171108144537_model_hmt_org_structure.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            "Office of Financial Sanctions Implementation",
+            "Payment Systems Regulator",
+            "Royal Mint",
+            "Royal Mint Advisory Committee",
+            "NS&I",
+            "The Crown Estate",
+            "UK Debt Management Office",
+            "UK Government Investments"
+        ],
+        ukgi => ["Government Corporate Finance Profession"]
```
Comment (h-lame) [inline db/migrate/20171108144537_model_hmt_org_structure.rb]: Missing `UK Financial Investments Limited` as a child here.

File: db/migrate/20171108150320_model_ukti_org_structure.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+class ModelUktiOrgStructure < ActiveRecord::Migration
+  def up
+    ukti = Organisation.find_by(name: "UK Trade & Investment")
+
+    child_org_names = [
+        "UKTI Life Sciences Organisation",
+        "Regeneration Investment Organisation",
+        "Financial Services Organisation"
```
Comment (h-lame) [inline db/migrate/20171108150320_model_ukti_org_structure.rb]: Missing `UKTI Education` as a child here.

Comment (elenatanasoiu) [reply db/migrate/20171108150320_model_ukti_org_structure.rb]: Added

File: db/migrate/20171108144537_model_hmt_org_structure.rb (hunk 1 above)
Comment (elenatanasoiu) [reply db/migrate/20171108144537_model_hmt_org_structure.rb]: Added

**Comment by h-lame** [comment]
Approved, but can we hold off on merging until I've finished the testing on integratio of #560?

**Pull request merged**